tools/integration-tests/tester/tests/value/value_test.go

```diff
@@ -0,0 +1,121 @@
+package autopeering
+
+import (
+	"testing"
+	"time"
+
+	"github.com/iotaledger/goshimmer/tools/integration-tests/tester/tests"
+	"github.com/stretchr/testify/require"
+)
+
+// TestValueIotaPersistence issues messages on random peers, restarts them and checks for persistence after restart.
+func TestValueIotaPersistence(t *testing.T) {
+	n, err := f.CreateNetwork("valueIota_TestPersistence", 4, 2)
+	require.NoError(t, err)
+	defer tests.ShutdownNetwork(t, n)
+
+	// wait for peers to change their state to synchronized
+	time.Sleep(5 * time.Second)
+
+	// master node sends funds to all peers in the network
+	txIds, addrBalance := tests.SendValueMessagesOnFaucet(t, n.Peers())
+
+	// wait for messages to be gossiped
+	time.Sleep(10 * time.Second)
```

Comment:
Currently a `transaction` becomes confirmed if a node hasn't seen a conflict for `2*networkdelay`.

I think in order to make sure that this is the case it should wait for at least this amount of time. Because in line 27 (`tests.CheckTransactions`) we check for the `confirmed` state on every node.